src/utils.js

```diff
@@ -5,10 +5,17 @@ function* simpleGenerator() {
     yield n++;
 }
 
-let simpleCounter = simpleGenerator();
+
+const simpleCounter = simpleGenerator();
+
+let currentId = 0;
+export function getCurrentId (){
```

Comment:
Where is this exported function used?

Reply:
That was already in the library, it's green because it got moved to a new location.

Reply:
Actually I was using that for debugging, it's getting removed.